jdaviz/configs/specviz/helper.py

```diff
@@ -23,6 +25,18 @@ def __init__(self, *args, **kwargs):
         self.app.hub.subscribe(self, RedshiftMessage,
                                handler=self._redshift_listener)
 
+        self.app.session.hub.subscribe(self, SubsetMessage,
```

Comment:
Why does this use ``app.session`` instead of just ``app``? (which is different from the redshift message)